### ISSUE TITLE
ImportError: cannot import name ReferencePattern

### ISSUE THREAD

**Issue by simevo** [jandecaluwe/urubu#81]
on debian buster:
```
python --version
Python 2.7.15+
```
to reproduce:
```
cd /tmp
mkdir q
virtualenv q
source q/bin/activate
pip install urubu
urubu build
```
trace:
```
Traceback (most recent call last):
  File "/tmp/q/bin/urubu", line 6, in <module>
    from urubu.main import main
  File "/tmp/q/local/lib/python2.7/site-packages/urubu/main.py", line 25, in <module>
    from urubu import project
  File "/tmp/q/local/lib/python2.7/site-packages/urubu/project.py", line 33, in <module>
    from urubu import readers, processors
  File "/tmp/q/local/lib/python2.7/site-packages/urubu/processors.py", line 31, in <module>
    from urubu import md_extensions
  File "/tmp/q/local/lib/python2.7/site-packages/urubu/md_extensions.py", line 31, in <module>
    from markdown.inlinepatterns import ReferencePattern, REFERENCE_RE, SHORT_REF_RE
ImportError: cannot import name ReferencePattern
```

**Comment by simevo** [comment]
here's the fix:
```
pip uninstall Markdown
pip install 'Markdown<3.0'
```
I guess you should put this in your pip package's `setup.py`

**Comment by LorenzoAncora** [comment]
Fixed by #87
